### PR TITLE
LPS-111561 Responsive for Collection Display Fragment

### PR DESCRIPTION
**Pull request by veroglez**
https://issues.liferay.com/browse/LPS-111561

@ealonso
@veroglez
cc @p2kmgcl

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/collection-general-panel/FeedbackMessage.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayIcon from '@clayui/icon';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export function FeedbackMessage({message}) {
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/collection-general-panel/FeedbackMessage.js:19]: Maybe we can move this component to "common" directory, it is quite generic :3

**Comment by veroglez** [comment]
ci:forward

**Comment by veroglez** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/7691

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**